sky/skylet/ray_patches/command_runner.py.patch

```diff
@@ -0,0 +1,4 @@
+330c330
+<                     "ControlPersist": "10s",
+---
+>                     "ControlPersist": "300s",
```

Comment:
Just curious, if we increase the number here, will it make the situation where `ssh` fails to connect to the cluster, hangs for a long time?

Reply:
I don't believe so. The fail-early params are 

https://github.com/ray-project/ray/blob/releases/1.13.0/python/ray/autoscaler/_private/command_runner.py#L320-L323

`ControlPersist` is an optimization - how long the ControlPath is in effect, I believe.

Reply:
Thanks for the explanation! Just found [another explanation](https://docs.rackspace.com/blog/speeding-up-ssh-session-creation/) of this config as well, place it here for a reference. ;)
We did previously observe some unexpected `ssh connection closed` during the `sky launch` (such as a 255 return code during rsyncing the task YAML to the spot controller), which may be relevant to the `ControlPersist` config for our ssh.
> ControlPath ~/.ssh/sockets/%r@%h-%p
> ControlPath is a specification of where to create the control socket on your filesystem. If you use the value provided here, make sure to manually mkdir ~/.ssh/sockets. At any time, you can manually remove the control socket (using plain-old rm), and the next ssh invocation will establish a new connection. This is especially useful if you’ve recently reopened your laptop and your SSH connections haven’t yet figured out that the server terminated them (especially prevalent if you’re connecting through a VPN).
> 
> ControlPersist 600
> Without ControlPersist, once the first SSH session you open is closed, all other sessions on that connection are closed as well. This can lead to a variety of surprising behavior: if you exit the initial SSH shell while other sessions are sharing the connection, the process will just hang. If you send it a Ctrl-C, all of those other sessions will be abruptly terminated.
> 
> In contrast, with ControlPersist set, the master connection will remain open for the specified number of seconds after your last SSH session on that connection has exited. This is especially useful for tasks such as pushing to Github, which occur frequently but where any one session is short-lived.